src/contact/dto/update-contact.dto.ts

```diff
@@ -0,0 +1,10 @@
+import { IsString, IsUUID } from 'class-validator';
+
+export class UpdateContactDto {
+  @IsUUID()
+  _id: string;
```

Comment:
Add blank line after each property definition

Reply:
the blank line was added